src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -4174,3 +4174,72 @@ def test_aks_create_with_apiserver_vnet_integration(self, resource_group, resour
         self.cmd(cmd, checks=[
             self.is_empty(),
         ])
+
+    def test_aks_draft_with_helm(self):
```

Comment:
It's ok to check-in the code first if you have a strict timeline for this feature. But don't forget to add the test cases later.

In order to avoid concurrent execution errors, the command (or in the test case) should support downloading the required binary to a temporary directory instead of a uniform path.

Besides, if the case would download a binary, better mark it as @live_only, otherwise the recording file would include the binary (the size of [test_aks_draft_with_helm.yaml](https://github.com/Azure/azure-cli-extensions/pull/4651/files#diff-4d1c6742e67dcee8c5df01fec4041cd3dae3d180bc3eebf04387b0739619878d) is 653,572 ), which is unnecessary. The case would be executed periodically in cli runner to make sure the command is available.

Reply:
@FumingZhang Once the test passes can you merge this?